src/server/middleware/createRequestStore.js

```diff
@@ -58,6 +59,9 @@ export default function createRequestStore(
         extraThunkArguments: { fetchClient },
       });
 
+      // re-set the client config before rendering the html fragment/document
+      store.dispatch(setConfig(clientConfig));
```

Comment:
this would mean that the client config is used for the server side render and negates the purpose of having a server config